prow/cmd/deck/static/script.js

```diff
@@ -61,12 +61,229 @@ function redrawOptions(opts) {
     addOptions(ss, "state");
 };
 
+function adjustScroll(el) {
+    var parent = el.parentElement;
+    var parentRect = parent.getBoundingClientRect();
+    var elRect = el.getBoundingClientRect();
+
+    if (elRect.top < parentRect.top) {
+        parent.scrollTop -= elRect.height;
+    } else if (elRect.top + elRect.height >= parentRect.top + parentRect.height) {
+        parent.scrollTop += elRect.height;
+    }
+ }
+
+function handleDownKey() {
+    var activeSearches =
+        document.getElementsByClassName("active-fuzzy-search");
+    if (activeSearches && activeSearches.length !== 1) return;
```

Comment:
Nit, but in my opinion prefer to make the truthy evaluation here more explicit -- are you worried about `activeSearches` being `null`? or `undefined`? Also, not putting brackets on single-line `if` only makes more more code changes down the line in case it needs to be edited, so maybe use them even in this case?

```js
if (activeSearches !== null && activeSearches.length !== 1 ) {
    return;
}
```

Reply:
done